openshift/deploy/vmaas_sync.yml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
```

Comment:
should this live here or in e2e-deploy?

Reply:
Eventually. We're using ocdeployer, and keeping up with the conventions set in `e2e-deploy` but for the time being we're keeping the templates here. Once we're comfortable with the set of components and we need to integrate into the platform deployment process, we can very simply move them.